config/manager/osl/sonataflow_builder_dockerfile.yaml

```diff
@@ -0,0 +1,25 @@
+FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-swf-builder-rhel8:latest AS builder
```

Comment:
we should be using a defined version, aka `1.30`

Reply:
Yes sure, is this needed also for the 'main'?